### DISCUSSION TITLE
Intersection Management between AGV's

### DISCUSSION THREAD

**Discussion by Christian-Goepfert** [VDA5050/VDA5050#110] [Q&A] [answered]
Hi VDA5050 core team,

my name is Christian Göpfert and I am working on a VDA5050 compliant software to control different AGV Vendors. During the work some questions came up:
Is intersection management already considered in the standard and if so how does it work? As far as I understood the text under subheadline 6.6 for creating orders, the agv plans a route and requests for permission from the central fleet management. If the permission is granted the agv drives until the next knot and plans the next steps of the route.
What happens if another agv crosses the way from a different vendor? Should the central fleet deny's the crossing or how should the bahavior according to the VDA5050 standard?

Thanks in advance for your help and best regards

Christian

**Comment by Pythocrates** [comment, marked as answer]
Hi, Christian,

the routes are planned by master control and then sent to the respective AGVs. For better coordination this can be done step by step by "releasing" just some nodes (the "base" in VDA-5050). You can always choose to release just the nodes up to an intersection to coordinate multiple AGVs crossing it. By applying some rule (e.g. right-hand rule or first come, first served) you can then establish an order in which you let the AGVs pass. That is up to the master control implementation and independent of the AGV types or vendors.

Best regards,
Martin

**Reply by MaximilianPenninger** [reply]
I agree with this answer. The idea of an VDA5050 "order" is not an order from bringing a good from A to B. A VDA5050-order is just a moving order that starts at a node and ends at a node. Your master control sends the agv/amr to the last node in front of the crossing - when the the robot is shortly to arrive the crossing (at the end of its 'base') it sends 'newBaseRequest" to the master control and if the crossing is vacant the master control can send an order update or a new order to the robot that includes passing the crossing.

**Comment by ConstantinUp** [comment]
Hi @Christian-Goepfert ,

@Pythocrates already answered the question about the current approach of the VDA 5050. It is important to note that VDA 5050 is not a recommendation for implementing traffic management, but rather a communication standard. 

With the addition of zones and the concept of path sharing, freely-navigating mobile robots that plan their own paths can now be better integrated using the VDA 5050. 
The master control is still sends the high-level task, e.g., go from your current position to position C, passing position B on the way. The vehicle will then plan its path and share it with the master control.
Zones can be defined to influence the mobile robots route planning. To avoid a collision, a RELEASE zone can be introduced, requiring the robot to request access from master control before entering.

If you are interested in this concept, feel free to check out the current dev branch dev/3.0.0 or PR #362. We welcome input on the proposed concept.

All the best,
Constantin
_Technical lead of the VDA 5050 core team_
